cpp/src/slice2matlab/Main.cpp

```diff
@@ -1978,23 +2028,29 @@ CodeVisitor::visitStructStart(const StructPtr& p)
     out.inc();
     vector<string> memberNames;
     DataMemberList convertMembers;
-    for (const auto& member : members)
+    for (const auto& q : members)
```

Comment:
I think member was a better name.

Reply:
I changed this name for consistency with other duplicates of the "same" loop.